src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -39,16 +39,13 @@
 #[AsCommand(name: 'lint:twig', description: 'Lint a Twig template and outputs encountered errors')]
 class LintCommand extends Command
 {
-    protected string|array $namePatterns;
-    private Environment $twig;
     private string $format;
 
-    public function __construct(Environment $twig, string|array $namePatterns = ['*.twig'])
-    {
+    public function __construct(
+        private Environment $twig,
+        private string|array $namePatterns = ['*.twig'],
```

Comment:
If you prefer, it can be restricted to `array`; this will not hurt.
```suggestion
        private array $namePatterns = ['*.twig'],
```

Reply:
updated